mozilla/fxa
# task(auth): Add third_party_accounts_set_password_complete metric

## Because

- We want to track when users set password after signing up with a third party

## This pull request

- Adds the third_party_accounts_set_password_complete metric.

## Issue that this pull request solves

Closes: FXA-9988

## Checklist

_Put an `x` in the boxes that apply_


## Other information (Optional)

To manually test

- Signup with using google (or apple)
- On the settings page set your password
- Observe that metric is emitted (e.g. `yarn pm2 log auth | grep third_party_auth_set_password`)
- Change password again, and validate that metric isn't emitted.
